galleries/examples/shapes_and_collections/hatchcolor_demo.py

```diff
@@ -0,0 +1,43 @@
+"""
+===========================
+Using hatchcolor in patches
+===========================
```

Comment:
```suggestion
================
Patch hatchcolor
================
```
or even just "Hatchcolor" are enough, we've recently stripped non-meaningful verbs ("Plotting ...", "Drawing ...", "Using ...") from example titles.